pkgs/applications/misc/organicmaps/default.nix

```diff
@@ -40,6 +41,14 @@ in stdenv.mkDerivation rec {
     fetchSubmodules = true;
   };
 
+  patches = [
+    # Fix for https://github.com/organicmaps/organicmaps/issues/7838
+    (fetchpatch {
+      url = "https://github.com/organicmaps/organicmaps/commit/6a2e3e9f9f8a745cc2759a8ac8e429f7bcd648b6.patch";
```

Comment:
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Use the merged commit, https://github.com/organicmaps/organicmaps/commit/1caf64e315c988cd8d5196c80be96efec6c74ccc

Reply:
Done